### ISSUE TITLE
build errors with Xcode 7.2.1

### ISSUE THREAD

**Issue by koenvanderdrift** [chriseidhof/github-issues#10]
Hoi Chris - just downloaded your project and I'm getting a bunch of build errors with Xcode 7.2.1. For a couple, Xcode gave a solution, but for the others I have no idea how to fix it. If you have a chance, would you mind looking into this?  

`github-issues-master/CoreDataStack.swift:107:67: Extra argument 'error' in call`
`github-issues-master/CoreDataStack.swift:140:51: Nil is not compatible with expected argument type '()'`
`github-issues-master/CoreDataStack.swift:175:46: Extra argument 'error' in call`
`github-issues-master/CoreDataStack.swift:140:13: Call can throw, but it is not marked with 'try' and the error is not handled`
`github-issues-master/GithubIssues/LoginViewController.swift:16:13: Variable 'vc' was never mutated; consider changing to 'let' constant`
`github-issues-master/GithubIssues/IssueEditViewController.swift:19:13: Variable 'vc' was never mutated; consider changing to 'let' constant`
`github-issues-master/GithubIssues/GithubAPI.swift:47:16: Immutable value 'dict' was never used; consider replacing with '_' or removing it`
`github-issues-master/GithubIssues/GithubAPI.swift:73:28: Missing argument labels 'method:requestParameters:parse:' in call`
`github-issues-master/GithubIssues/GithubAPI.swift:87:28: Missing argument labels 'method:requestParameters:parse:' in call`
`github-issues-master/GithubIssues/GithubAPI.swift:105:28: Missing argument labels 'method:requestParameters:parse:' in call`
`github-issues-master/GithubIssues/GithubAPI.swift:149:24: Missing argument labels 'method:requestParameters:parse:' in call`
`github-issues-master/GithubIssues/GithubAPI.swift:160:24: Missing argument labels 'method:requestParameters:parse:' in call`
`github-issues-master/GithubIssues/GithubAPI.swift:165:13: Variable 'result' was never mutated; consider changing to 'let' constant`
`github-issues-master/GithubIssues/GithubAPI.swift:181:15: Missing argument labels 'baseURL:resource:failure:progress:completion:' in call
FunctionalViewControllers Group`
`github-issues-master/FunctionalViewControllers/FunctionalViewControllers/TableView.swift:51:95: Closure parameter prior to parameters with default arguments will not be treated as a trailing closure`
`github-issues-master/FunctionalViewControllers/FunctionalViewControllers/APIClient.swift:71:39: Immutable value 'statusCode' was never used; consider replacing with '_' or removing it`
`github-issues-master/FunctionalViewControllers/FunctionalViewControllers/APIClient.swift:72:34: Immutable value 'errors' was never used; consider replacing with '_' or removing it`

**Comment by chriseidhof** [comment]
Yeah it's a really old project! This is because it was built before Swift has error handling. Feel free to give it a try, and let me know if you are able to fix it =)

**Comment by koenvanderdrift** [comment]
Ok, I'll see what I can find out - I'm still very new to Swift (coming from ObjC), so I cannot promise anything :)
